lms/djangoapps/django_comment_client/tests/utils.py

```diff
@@ -76,6 +76,15 @@ def setUp(self):
                     scheme_id='cohort'
                 )
             ],
+            grading_policy={
```

Comment:
I'd like to see this class factored out of `django_comment_client` (I know, I know, I wrote it...) into perhaps `common/djangoapps/util/testing.py`? Or openedx?